dds_cli/maintenance_manager.py

```diff
@@ -53,7 +53,7 @@ def __init__(
 
         # Only methods "on" and "off" can use the Maintenance class
         if self.method not in ["on", "off"]:
-            raise dds_cli.exceptions.AuthenticationError(f"Unauthorized method: '{self.method}'")
+            raise dds_cli.exceptions.InvalidMethodError(f"Unauthorized method: '{self.method}'")
```

Comment:
Also add `dds_cli.exceptions.InvalidMethodError,` to `__main__.py` in the `except` in command `dds maintenance` so that the exception is displayed in a better way if the exception is raised.